troposphere/static/css/app/app.scss

```diff
@@ -31,7 +31,7 @@ $icon-font-path: "~bootstrap-sass/assets/fonts/bootstrap/";
 // Keep Modal footer at bottom on min-height modals
 #modal {
   .modal-footer {
-    position: absolute;
+    position: relative;
```

Comment:
This is for showing the login button correctly. A temporary solution: https://github.com/cyverse/troposphere/issues/630